js/main.js

```diff
@@ -0,0 +1,504 @@
+let maxBoardSize = 520;													//максимальний розмір дошки у px, висота або ширина
```

Comment:
Bad practice to keep all of functionality in one main file. You should separate logic to diferent files if it's possible